auth/credentials/internal/externalaccount/externalaccount.go

```diff
@@ -161,12 +161,12 @@ func (o *Options) validate() error {
 	if o.CredentialSource != nil {
 		count++
 	}
-	// if o.SubjectTokenSupplier != nil {
-	// 	count++
-	// }
-	// if o.AwsSecurityCredentialsSupplier != nil {
-	// 	count++
-	// }
+	if o.SubjectTokenProvider != nil {
+		count++
+	}
+	if o.AwsSecurityCredentialsProvider != nil {
+		count++
+	}
 	if count == 0 {
 		return fmt.Errorf("externalaccount: one of CredentialSource, SubjectTokenSupplier, or AwsSecurityCredentialsSupplier must be set")
```

Comment:
Be sure to convert usages of "supplier" to the new name, both here and in all doc comments.

Reply:
Thanks, missed this one!